.npmignore

```diff
@@ -0,0 +1,5 @@
+node_modules
+.circleci
+.dist
+.envrc
+.envrc.default
```

Comment:
Actually .envrc.default is ok to have in repo I think.

Reply:
I agree, but is it ok in npm package?

Reply:
(this file is .npmignore, not .gitignore) @mkucharz